assets/sass/pages/_docs.scss

```diff
@@ -1151,25 +1151,54 @@
     .docs-wrap {}
 
     .texts-container {
-        h1, h5 {
-            margin-top: 5px;
-            padding-bottom: 0;
-
-            color: $c-blue-dark1;
-            font-family: 'Poppins';
+        h1 {
+            margin-bottom: 40px;
             font-size: 40px;
-            font-weight: 200;
+            margin-bottom: 20px;
+            font-weight: 400;
+            margin-top: 0;
+            line-height: 1.4;
+        }
+      
+        h2 {
+            line-height: 1.4;
+            font-size: 32px;
+            font-weight: 400;
+            margin-bottom: 20px;
+            margin-top: 0;
+        }
+
+        h3 {
+            font-size: 25px;
+            font-size: 28px;
+            font-weight: 400;
+            margin-bottom: 20px;
+            margin-top: 0;
         }
 
-        h3,
         h4 {
-            margin-bottom: 15px;
-            padding-top: 5px;
+            font-size: 20px;
+            font-size: 24px;
+            font-weight: 400;
+            margin-bottom: 10px;
+            margin-top: 0;
+        }
 
-            color: $c-blue-dark1;
-            font-family: 'Poppins';
-            font-size: 25px;
+        h5 {
+            margin-bottom: 1em;
+            font-size: 20px;
+            font-size: 18px;
+            font-weight: 400;
+            font-weight: bold;
```

Comment:
`font-weight` should only be defined once. I prefer using the weight numbers rather than light, normal, bold, bolder as it is easier to understand and tie back to the weight of the font family which is generally listed that way.

Reply:
used only `font-weight: 600`